docs/user-guide/glossary.md

```diff
@@ -61,13 +61,25 @@ components](#component-category) together. It can be connected to the public
 [grid](#grid), or be completely isolated, in which case it is known as an
 island.
 
+Components can be grouped into [assets](#Assets) and [devices](#Devices).
+Assets are core components like generators or storage systems that are crucial from a business perspective,
+whereas devices are the supporting infrastructure essential for the functionality of these assets.
+
 ### Component Category
 
 The category [components](#component) of a [microgrid](#microgrid) belong to.
 
 [Components](#component) of the same category have the same characteristics
 (for example offer the same set of [metrics](#metric)).
 
+Examples of categories are meters, inverters, batteries and sensors.
```

Comment:
This doesn't match the microgrid API (or won't match it when we migrate to v0.17), sensors are not a component category anymore (it was in previous versions). Actually the [common API](https://github.com/frequenz-floss/frequenz-api-common/blob/29f9945bde7131c3851a1e94a6c19c509ed93806/proto/frequenz/api/common/v1/microgrid/components/components.proto#L26), so it doesn't match the dispatch API either, and probably the reporting API either.

Not sure if you are aware of this but still conceptually we want to say sensors are components, but I think it will be confusing if then you get components from the microgrid API and there are no sensors there.

Reply:
Fixed.